rclpy/rclpy/__init__.py

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import rclpy._rclpy__rmw_opensplice_cpp as _rclpy
+import os
 
+from rclpy import selector
 from rclpy.node import Node
 
+_rclpy = None
+
 
 def init(args):
+    global _rclpy
```

Comment:
It looks like the other functions do not use `global _rclpy`. Can you either remove it here or add it to the other functions?